README.md

```diff
@@ -1,24 +1,85 @@
 # `Gjallarhorn`
 
-Gjallarhorn is a tool for Brawlhalla community tournament runners to generate
-properly formatted JSON for Twitch overlays. This supports overlays for Game,
-Players, Queue, Ticker, Casters, and Lower Thirds. It pulls data from start.gg
-as well as Brawlhalla's Stats API.
+Gjallarhorn is a tool for Brawlhalla community tournament runners to generate properly formatted JSON for Twitch overlays. Gjallarhorn can generate JSON for multiple use cases, including tickers, casters, lower thirds, queues, players, and games. It pulls data from start.gg as well as Brawlhalla's Stats API.
 
-## Usage
+## Getting Started
+
+You will need a Start.gg API key to use Gjallarhorn. If you don't have one already, get one at https://developer.start.gg/docs/authentication.
```

Comment:
probably doesn't matter but sgg's brand guidelines say we should only ever write "start.gg"
https://help.start.gg/en/articles/1716774-start-gg-brand-guidelines

Reply:
This is a good catch. I went ahead and standardized that usage in the doc.